Makefile

```
@@ -1,5 +1,5 @@
-CRYSTAL_BIN ?= $(shell which crystal)
-SHARDS_BIN ?= $(shell which shards)
+CRYSTAL_BIN ?= $(which crystal)
+SHARDS_BIN ?= $(which shards)
```

Unfortunately, that broke the CI builds on non freebsd env. I think it can be a bit simplified. 

```suggestion
CRYSTAL_BIN ?= crystal
SHARDS_BIN ?= shards
```